lib/worker.js

```diff
@@ -15,15 +15,18 @@ const eslintPathLocal = Path.join(FS.realpathSync(Path.join(__dirname, '..')), '
 let eslint = null
 let prefixPath = null
 
-function find(startDir, name) {
-  let filePath
+function find(startDir, names) {
```

Comment:
This function is used in several places, not all of them pass array, so you should add something like
```js
if (typeof names === 'string') {
  names = [names]
}
```